File: All_File/C/bubblesort.c
#include<stdio.h>
int main(){
int a[100],j,temp,n,i;
printf("Enter How many number:\n");
scanf("%d",&n);//4
printf("Enter the element:\n");
for(i=0;i<n;i++){
	scanf("%d",&a[i]);
}
for(i=0;i<n-1;i++){//number of passes
	for(j=0;j<n-1-i;j++){/// number of comperssion
		if(a[j]>a[j+1]){//jodi condition true hoy tahole baki ta hobe nahole abar for loop a  jabe
			temp=a[j];//temp ta a[j] ar man k hold korbe
			a[j]=a[j+1];//a[j]ta a tar pore ar  man ta hobe 
			a[j+1]=temp; //r temp ta seta kai hold korbe
		}
	}
}
for(i=0;i<n;i++)	
printf(" sorting are:%d",a[i]);
}
//#include<stdio.h>
//#include<stdlib.h>
//void parray(int* a,int n){
//	int i1;
//	for(i1=0;i1<n;i1++){
//		printf("%d",&a[i1]);
//	}
//	for(i=0;i<n;++i)
//	printf("%d",a[i]);
//}
//
//
//
//int main(){
//	int a[100],i,n;
//	printf("Enter t How many number:\n");
//	scanf("%d",&n);
//	printf("Enter the element:\n");
//	for(i=0;i<n;i++){
//		scanf("%d",&a[i]);
//	}
//	
//}

File: All_File/Java_oop_practice_for_Job_Interview/bubblesort.java
public class bubblesort {
    public static void main(String[] args) {
        
        // int a[]={7,3,5,4};
        // for(int i=0;i<a.length;i++){
        //     for(int j=1;j<a.length-1;i++){
        //         if(a[i]>a[j]){
        //             int temp=a[i];
        //             a[i]=a[j];
        //             a[j]=temp;
        //         }
        //     }
        // }
        // for(int i=0;i<a.length;i++){
        //     System.out.println(a[i]);
        // }
        int a[]={5,4,8,2,7,1};
		for(int i=1;i<a.length;i++){
          for(int j=0;j<a.length;j++){
          	if (a[j]>a[j+1]) {
          		int temp=a[j];
          		a[j]=a[j+1];
          		a[j]=temp;
          	}
        }
          
          for(i=0;i<a.length;i++)
          System.out.println(a[i]);
  }
    }
}
